src/Tizen.NUI/src/internal/Widget/WidgetImpl.cs

```diff
@@ -312,7 +312,10 @@ private void SwigDirectorOnCreate(string contentInfo, global::System.IntPtr wind
             {
                 ret = new Window(window, true);
             }
+
             OnCreate(contentInfo, ret);
+            ret.Dispose();
```

Comment:
To safety way, we must 'not' call Dispose here. I don't know detail logic for it but.. If the `ret` Window created by user-side, it can make unknown disposed.

Please check what I did

https://github.com/Samsung/TizenFX/pull/5374/files#diff-8cd06cf3f9b6472b4b266dd90e5e950851d14d7b85054d4f227bd5e1601e871fR196

Reply:
Or, see `Tizen.NUI.Extensions.GetInstanceSafely<Window>` API usage. It will be helpful to catch the memory leak.